code/modules/mob/living/carbon/human/combat.dm

```diff
@@ -29,6 +29,16 @@
 /mob/living/carbon/human/disarm_mob(mob/living/target)
 	add_logs(src, target, "disarmed", admin = (src.ckey && target.ckey) ? TRUE : FALSE) //Only add this to the server logs if both mobs were controlled by player
 
+	var/datum/organ/external/S = target.get_organ(src.zone_sel.selecting)
+	if(src.zone_sel.selecting == "mouth" && !(S.status & ORGAN_DESTROYED) && ishuman(target))
```

Comment:
Somewhere before the `S.status` check, there should be an `istype(S)` check

Honestly just checking for the existence of S would be sufficient, but `istype(S)` would encompass any wacky antics Schrödinger's Oldcode could theoretically throw at it